.github/release-drafter-config.yml

```diff
@@ -0,0 +1,48 @@
+name-template: 'v$RESOLVED_VERSION 🌈'
+tag-template: 'v$RESOLVED_VERSION'
+categories:
+  - title: '🚀 Features'
+    labels:
+      - 'enhancement'
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'bug'
+autolabeler:
+  - label: 'documentation'
+    files:
+      - '*.md'
```

Comment:
It's a good idea to include all the files in the docs directory.

Reply:
I think that for the autolabelel purpose `.md` plus `.svg` should be good enough. Currently, our docs consist of only md and svg files

Reply:
Hope you will remember that :) 
Yes, it is good enough for now, but later there will be files with other extension.